### PR TITLE
Make shared-fs fio job file work similar to non-shared

### LINKED ISSUE #1255: numjobs not creating more jobs in pbench-fio with shared-fs job file
With pbench-agent test RPM v0.61, I am able to test with the shared-fs job file, but when using --numjobs=4 in pbench-fio, I don't believe this is translating into 4 jobs per client.  Results show only log files (bs, iops, clat, etc) for 1 job per client, and there is no "--numjobs=4" in the fio.cmd file that pbench-fio creates.

@portante @bengland2 @ndokos FYI

### PR DESCRIPTION
**Pull request by portante**
Fixes #1255.

### CODE REVIEW

**Comment by portante** [comment]
@bengland2, @atheurer, can you two please take a look to see if this change makes sense?

**Comment by ndokos** [comment]
@atheurer: in case you missed it, the 0.61-2 RPM available in the copr-pbench-test repo includes this fix and can be used for testing.

**Comment by atheurer** [comment]
Can anyone explain what is going on here:
<pre>
[root@perf84 ~]# pbench-fio --install
[root@perf84 ~]# rpm -qa | grep pbench-fio
pbench-fio-3.12-1.x86_64

[root@perf84 ~]# yum --enablerepo=copr-pbench-test update pbench-agent
Loaded plugins: langpacks
Resolving Dependencies
--> Running transaction check
---> Package pbench-agent.noarch 0:0.60-1gc9e89f8f will be updated
---> Package pbench-agent.noarch 0:0.61-2g1e887664 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

============================================================================================================================================
 Package                          Arch                       Version                             Repository                            Size
============================================================================================================================================
Updating:
 pbench-agent                     noarch                     0.61-2g1e887664                     copr-pbench-test                     263 k

Transaction Summary
============================================================================================================================================
Upgrade  1 Package

Total download size: 263 k
Is this ok [y/d/N]: y
Downloading packages:
No Presto metadata available for copr-pbench-test
pbench-agent-0.61-2g1e887664.noarch.rpm                                                                              | 263 kB  00:00:01
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : pbench-agent-0.61-2g1e887664.noarch                                                                                      1/2
  Cleanup    : pbench-agent-0.60-1gc9e89f8f.noarch                                                                                      2/2
  Verifying  : pbench-agent-0.61-2g1e887664.noarch                                                                                      1/2
  Verifying  : pbench-agent-0.60-1gc9e89f8f.noarch                                                                                      2/2

Updated:
  pbench-agent.noarch 0:0.61-2g1e887664

Complete!

</pre>log out and back in<pre>

[root@perf84 ~]# pbench-fio --install
Traceback (most recent call last):
  File "/usr/bin/getconf.py", line 65, in <module>
    status = main(conf, args, opts)
  File "/usr/bin/getconf.py", line 31, in main
    configtools.print_list(configtools.get_list(conf.get(sec, option)), sep)
  File "/usr/lib64/python2.7/ConfigParser.py", line 623, in get
    return self._interpolate(section, option, value, d)
  File "/usr/lib64/python2.7/ConfigParser.py", line 691, in _interpolate
    self._interpolate_some(option, L, rawval, section, vars, 1)
  File "/usr/lib64/python2.7/ConfigParser.py", line 723, in _interpolate_some
    option, section, rest, var)
ConfigParser.InterpolationMissingOptionError: Bad value substitution:
	section: [pbench-agent]
	option : install-dir
	key    : pbench_install_dir
	rawval :

[error][2019-07-12T20:57:42.269049896] [check_install_rpm] the installation of pbench-fio-3.3 failed
</pre>

**Comment by atheurer** [comment]
I can confirm the --numjobs is now working.  Not sure how the install version for fio is getting changed, but I do see this in master: https://github.com/distributed-system-analysis/pbench/blob/63be766411fbe60a839f60fb99fb5ead33fd8f1f/agent/config/pbench-agent-default.cfg#L30  ...which makes no sense to me.  Why do we have a pbench version (3.3) which we do not have an RPM for?

**Comment by ndokos** [comment]
Re: the pbench-fio version - I'll open an issue and get it fixed: it should be 3.12.

**Comment by ndokos** [comment]
@atheurer: The `pbench_install_dir` issue that caused `getconf.py` to fail has been fixed in the internal config files. The pbench-fio version problem has been fixed by PR #1266. None of this was affecting 0.60 installations but it would have bit us going forward. Thanks for pointing them out!

File: agent/bench-scripts/templates/fio-shared-fs.job [hunk 1]
```diff
@@ -24,10 +25,11 @@ log_hist_msec = 10000
 # log_hist_coarseness = 4 # 76 bins
 
 [TheJob]
-runtime = $runtime
 directory = $target
 rw = $@
-size = $size
+size = 4096M
 # if you want to specify more than 1 process per fio client host/pod,
 # uncomment this next line and add --numjobs=K to your pbench-fio command
```
[the remark this answers is not in the document]
Comment (portante) [reply agent/bench-scripts/templates/fio-shared-fs.job]: Changed in latest commit.

**Pull request merged**